### PR TITLE
feat(link-list-section): Web component created

### PR DESCRIPTION
**Pull request by RaphaelAmadeu**
### Related Ticket(s)

#3376
### Description

Created the web-component version of `LinkListSection` following it's design and functional specs.

### CODE REVIEW

File: packages/web-components/src/components/link-list/link-list.ts [hunk 1]
```diff
@@ -87,6 +87,10 @@ class DDSLinkList extends StableSelectorMixin(LitElement) {
     window.removeEventListener('resize', this._handleResize);
   }
 
+  updated() {
+    sameHeight(this.childItems, 'md');
+  }
+
```
Comment (asudoh) [inline packages/web-components/src/components/link-list/link-list.ts:93]: Let's leave this not running for now - We are trying to determine what to do with `sameWidth()`/`sameHeight()` as a whole.

```suggestion
```

**Review by asudoh — COMMENTED** [review]
LGTM basically, just one thing - Thanks @RaphaelAmadeu!

**Review by asudoh — APPROVED** [review]
Thanks @RaphaelAmadeu for the update!

**Review by jeffchew — CHANGES_REQUESTED** [review]
Visually looks broken to me, can you take a look @RaphaelAmadeu ?

Screen Shot 2020-12-08 at 11 23 48 AM

**Comment by RaphaelAmadeu** [comment]
> Visually looks broken to me, can you take a look @RaphaelAmadeu ?
> 
> Screen Shot 2020-12-08 at 11 23 48 AM
> 
> Also, the props table isn't loading:
> 
> Screen Shot 2020-12-08 at 11 25 45 AM

I've addressed these issues on my latest commit, thanks!

**Comment by jeffchew** [comment]
Thanks @RaphaelAmadeu ! can you check the web components ci-checks? probably snapshots...

**Review by jeffchew — APPROVED** [review]
Thank you @RaphaelAmadeu !

**Pull request merged**